core/src/main/java/MyCustomSerializer.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+
+//import tech.beshu.ror.AuditLogContext;
+//import tech.beshu.ror.requestcontext.AuditLogSerializer;
+//
+//import java.util.HashMap;
+//import java.util.Map;
+//
+//public class MyCustomSerializer implements AuditLogSerializer {
```

Comment:
It looks like a leftover - this class shouldn't be included.

Reply:
Yeah I left this here as an example of how to write a serializer until I make the docs.